src/services/twilio-api/api-command-runner.js

```diff
@@ -44,7 +44,7 @@ class ApiCommandRunner {
 
     // If there were any errors validating the flag values, log them by flag
     // key and throw.
-    if (Object.keys(flagErrors).length > 0) {
+    if (Object.keys(flagErrors).length > 0 && !this.flagValues['skip-parameter-validation']) {
```

Comment:
Is it beneficial to log the errors but not throw?

Reply:
Was on vacation @childish-sambino ... it wouldn't be beneficial for the tricorder use-case.